frontend/ui/widget/menu.lua

```diff
@@ -593,20 +593,68 @@ function Menu:init()
     self.page_info_first_chev:hide()
     self.page_info_last_chev:hide()
 
+    local title_goto, type_goto, hint_func
+    local buttons = {
+        {
+            {
+                text = _("Cancel"),
+                callback = function()
+                    self.page_info_text:closeInputDialog()
+                end,
+            },
+            {
+                text = _("Go to page"),
+                is_enter_default = true,
+                callback = function()
+                    local page = tonumber(self.page_info_text.input_dialog:getInputText())
+                    if page and page >= 1 and page <= self.page_num then
+                        self:onGotoPage(page)
+                    end
+                    self.page_info_text:closeInputDialog()
+                end,
+            },
+        },
+    }
+
+    if self.goto_letter then
+        title_goto = _("Input page number or letter")
+        type_goto = "string"
+        hint_func = function()
+            return string.format("(1 - %s) or (a - z)", self.page_num)
+        end
+        table.insert(buttons[1], {
+            text = _("Go to letter"),
+            is_enter_default = true,
+            callback = function()
+                logger.info(self.item_table)
```

Comment:
You may want to remove this `logger.info` before merge (to not fill our devices' crash.log :)

Reply:
Of course :)